sdk/translation/azure-ai-translation-document/README.md

```diff
@@ -86,6 +86,35 @@ credential = AzureKeyCredential("<api_key>")
 document_translation_client = DocumentTranslationClient(endpoint, credential)
 ```
 
+#### Create the client with an Azure Active Directory credential
```

Comment:
out of curiosity, do you know if there are any service docs about supporting AAD?